Observer/SendInvoiceMail.php

```diff
@@ -74,12 +80,22 @@ public function execute(Observer $observer)
         if (!$invoice->getEmailSent() && $invoice->getIsPaid() && $canCapture && $sendInvoiceEmail) {
             $invoice->save();
             $this->logging->addDebug(__METHOD__ . '|10|sendinvoiceemail');
-            $orderBaseShippingAmount = $invoice->getOrder()->getBaseShippingAmount();
+            $orderBaseShippingAmount = $order->getBaseShippingAmount();
             $this->invoiceSender->send($invoice, true);
-            if (($orderBaseShippingAmount > 0) && ($invoice->getOrder()->getBaseShippingAmount() == 0)) {
+            if (($orderBaseShippingAmount > 0) && ($order->getBaseShippingAmount() == 0)) {
                 $this->logging->addDebug(__METHOD__ . '|15|');
                 $invoice->getOrder()->setBaseShippingAmount($orderBaseShippingAmount);
             }
         }
+        if ($invoice->getIsPaid() && $canCapture) {
+            if (
+                ($payment->getMethod() == 'buckaroo_magento2_afterpay20')
```

Comment:
we could use the constant PAYMENT_METHOD_CODE instead of string.

Reply:
changed, thx